Source/WebCore/platform/text/WritingMode.h

```diff
@@ -75,66 +75,56 @@ constexpr inline BlockFlowDirection writingModeToBlockFlowDirection(WritingMode
     return BlockFlowDirection::TopToBottom;
 }
 
-constexpr unsigned makeTextFlowInitalizer(BlockFlowDirection blockFlow, TextDirection direction)
-{
-    return static_cast<unsigned>(blockFlow) << 1 | static_cast<unsigned>(direction);
-}
-
 // Define the text flow in terms of the writing mode and the text direction. The first
 // part is the block flow direction and the second part is the inline base direction.
-enum TextFlow {
-    InlineEastBlockSouth = makeTextFlowInitalizer(BlockFlowDirection::TopToBottom, TextDirection::LTR),
-    InlineWestBlockSouth = makeTextFlowInitalizer(BlockFlowDirection::TopToBottom, TextDirection::RTL),
-    InlineEastBlockNorth = makeTextFlowInitalizer(BlockFlowDirection::BottomToTop, TextDirection::LTR),
-    InlineWestBlockNorth = makeTextFlowInitalizer(BlockFlowDirection::BottomToTop, TextDirection::RTL),
-    InlineSouthBlockEast = makeTextFlowInitalizer(BlockFlowDirection::LeftToRight, TextDirection::LTR),
-    InlineSouthBlockWest = makeTextFlowInitalizer(BlockFlowDirection::LeftToRight, TextDirection::RTL),
-    InlineNorthBlockEast = makeTextFlowInitalizer(BlockFlowDirection::RightToLeft, TextDirection::LTR),
-    InlineNorthBlockWest = makeTextFlowInitalizer(BlockFlowDirection::RightToLeft, TextDirection::RTL)
-};
+struct TextFlow {
+    BlockFlowDirection blockFlowDirection;
```

Comment:
Since this is a TextFlow object, maybe just `blockDirection` is enough?

Reply:
(Shorter words here make things less noisy in the complex conditionals these often end up in, and also shorter helps avoid awkward wrapping in such conditionals.)